src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Types/IContentFieldProvider.cs

```diff
@@ -8,5 +9,15 @@ public interface IContentFieldProvider
         FieldType GetField(ISchema schema, ContentPartFieldDefinition field, string namedPartTechnicalName, string customFieldName = null);
 
         bool HasField(ISchema schema, ContentPartFieldDefinition field);
+
+        FieldTypeIndexDescriptor GetFieldIndex(ContentPartFieldDefinition field);
```

Comment:
We have only recently added `ISchema` to the other methods in this interface. Should the new methods also have access to it?

Reply:
@gvkries, I don't think it will be needed here, also those methods are used inside `DynamicContentFieldsIndexAliasProvider` to dynamically register content field indexes which can't make ISchema available